File: 2015/src/test/java/Day3Test.java
import nl.sbmf21.aoc15.Aoc;
import nl.sbmf21.aoc15.days.Day3;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class Day3Test {

    private final Day3 day = new Day3(new Aoc(), 3);

    @Test
    public void testPart1Output() {
        assertEquals(2081, (int) day.part1());
    }

    @Test
    public void testPart2Output() {
        assertEquals(2341, (int) day.part2());
    }
}

File: 2020/kotlin/src/test/kotlin/Day3Test.kt
import nl.sbmf21.aoc20.Aoc
import nl.sbmf21.aoc20.days.Day3
import org.junit.Test
import kotlin.test.assertEquals

class Day3Test {

    private var day = Day3(Aoc(), 3)

    @Test
    fun testPart1Output() = assertEquals(191, day.part1())

    @Test
    fun testPart2Output() = assertEquals(1478615040, day.part2())
}
